internal/integrationtest/main/main_test.go

```diff
@@ -69,8 +70,7 @@ func TestVersion(t *testing.T) {
 	// Checks if Application's value is arduino-cli
 	require.Equal(t, jsonMap["Application"], "arduino-cli")
 
-	// Checks if VersionString's value is git-snapshot,
-	// nightly or a valid semantic versioning
+	// Checks if VersionString's value is git-snapshot, nightly or a valid semantic versioning
```

Comment:
This one should be part of https://github.com/arduino/arduino-cli/commit/095b590a45fa9e77f573af306d84da6da03bd0d6